de.metas.swat/de.metas.swat.base/src/main/java/de/metas/ordercandidate/api/IOLCandEffectiveValuesBL.java

```diff
@@ -251,5 +251,50 @@
 	 */
```

Comment:
Please check out and sync a the javadocs of the methods you changed. E.g. the javadoc for IOLCandEffectiveValuesBL.getDropShip_BPartner_Effective_ID() is not currect (anymore?)